crates/consensus/common/src/validation.rs

```diff
@@ -36,6 +36,51 @@ pub fn validate_header_base_fee(
     }
     Ok(())
 }
+// EIP-4895: Beacon chain push withdrawals as operations
+#[inline]
```

Comment:
please make this an actual doc comment (and same for other fns below), i.e. `///`